src/Symfony/Component/DomCrawler/Form.php

```diff
@@ -39,14 +39,14 @@ class Form extends Link implements \ArrayAccess
     /**
      * Constructor.
      *
-     * @param \DOMElement $node       A \DOMElement instance
+     * @param \DOMNode    $node       A \DOMNode instance
      * @param string      $currentUri The URI of the page where the form is embedded
      * @param string      $method     The method to use for the link (if null, it defaults to the method defined by the form)
      * @param string      $baseHref   The URI of the <base> used for relative links, but not for empty action
      *
      * @throws \LogicException if the node is not a button inside a form tag
      */
-    public function __construct(\DOMElement $node, $currentUri, $method = null, $baseHref = null)
+    public function __construct(\DOMNode $node, $currentUri, $method = null, $baseHref = null)
```

Comment:
This should not be changed back, as this class cannot work with anything else than a DOMElement (it triggers fatal errors for others)